### ISSUE TITLE
Websockets: do not "reconnect" after error

### ISSUE THREAD

**Issue by Amantel** [jaggedsoft/node-binance-api#510]
TLTR: I subscribe to 500 symbols 1h klines, some of them get `ESOCKETTIMEOUT`, I never get those symbols again. 

Note: Reconnect is in quotes, because I am not sure that this is reconnection issue or this should be approached from some other side. 

I have 500+ symbols I want to track on 1h interval. For that I use `binance.websockets.chart(fullSymbolsList, '1h', async (symbol, interval, chart) => { doSomething }` 
Sometimes (on a decent % of them) I receive errors in getSymbolKlineSnapshot function. That error is not used, but if you start to log it, you will see that it is there. And if we have an error, package treats data as  {}.
The error is `ESOCKETTIMEOUT`
After that I wait in the hope that I will get those symbols back on track, but I don't. 
Using binance graph interface I see that some activity is happening on those symbols (I use 1m interval to track that), so I should receive candles. But I don't. 
Additionally I used this demo https://2pd.github.io/ws/ to check double check and it does not get new updates after initial fail (I suspect it is a fail, because there is eternal loading symbol there) too. 
Thus, this may or may not be a problem with binance websockets. 
For me this means that I can't safely assume that by subscribing to my symbols list I will get all the symbols data and that's break the whole point of using websockets.

**Comment by patzero1** [comment]
I have the same problem, did you find any solution @Amantel ?

**Comment by Amantel** [comment]
Nope. I was digging for some time, but dropped the websockets in the end.

ср, 18 нояб. 2020 г., 15:11 patzero1 <notifications@github.com>:

**Comment by wearethefuture-org** [comment]
Same for me

**Comment by wearethefuture-org** [comment]
@Amantel @patzero1 just use keepAlive: true, I looked inside npm module.

const Binance = require('node-binance-api');
const binance = new Binance().options({
  APIKEY: apiKey,
  APISECRET: secretKey,
  keepAlive: true
});
